_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+- category: Server-side
+  items:
+    - title: Build AMIs
+      description: |
+        If you want to run your apps directly on EC2 Instances, you should package them as
+        <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html" target="_blank">Amazon Machine Images (AMIs)</a>
+        using a tool such as <a href="https://www.packer.io/" target="_blank">Packer</a>. Although we recommend Docker
+        for all stateful apps (see below), we recommend directly using AMIs and EC2 Instances for all stateful apps,
```

Comment:
for all ~stateful~ stateless apps (see below),...

Reply:
Fixed! https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/ac969266fa45512affe5434f70111b488f8c4eb8